src/python/pants/core/goals/test.py

```diff
@@ -224,6 +230,17 @@ def level(self) -> LogLevel:
             return LogLevel.DEBUG
         return LogLevel.INFO if self.exit_code == 0 else LogLevel.ERROR
 
+    def _simplified_output(self, v: bytes) -> str:
+        return self.output_simplifier.simplify(v.decode(errors="replace"))
+
+    @property
```

Comment:
Is it worth using @memoized_property on these?

Reply:
Probably doesn't hurt, will do!